data/json/recipes/recipe_food.json

```
@@ -2285,7 +2285,7 @@
     "charges": 1,
     "time": "1 h 30 m",
     "batch_time_factors": [ 50, 4 ],
-    "autolearn": true,
+    "book_learn": [ [ "baking_book", 3 ] ],
```

```suggestion
    "book_learn": [  [ "baking_book", 3 ], [ "cookbook_eatyrway", 5 ] ],
```
I think this fits the "boring/gross" food plan for eatyrway. It could go in a survival textbook too but they're already overloaded. We might consider making a survival cookbook.

---

Made a tiny pocket survival cookbook.  Also has no recipes added to it as of yet.